DuckDuckGo/AppDelegate.swift

```diff
@@ -191,12 +191,18 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
         // MARK: Sync initialisation
 
+#if DEBUG
+        let defaultEnvironment = ServerEnvironment.development
+#else
+        let defaultEnvironment = ServerEnvironment.production
+#endif
+
         let environment = ServerEnvironment(
             UserDefaultsWrapper(
                 key: .syncEnvironment,
                 defaultValue: ServerEnvironment.production.description
```

Comment:
This results in the default still being production. 

```suggestion
                defaultValue: defaultEnvironment.description
```